modules/vpc/outputs.tf

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "network" {
+  value       = google_compute_network.network
+  description = "The VPC resource being created"
+}
+
+output "network_name" {
+  value       = google_compute_network.network.name
+  description = "The name of the VPC being created"
+}
+
+output "network_self_link" {
+  value       = google_compute_network.network.self_link
+  description = "The URI of the VPC being created"
+}
+
+output "svpc_host_project_id" {
```

Comment:
This could just be project_id, no? (We can/should make it depend on the shared_vpc_host_project resource though.)

Reply:
Yeah seems logical. Updated

Reply:
Look like it still needs to be updated?